src/cmd/flux-job.c

```diff
@@ -69,6 +69,19 @@ static struct optparse_option list_opts[] =  {
     { .name = "suppress-header", .key = 's', .has_arg = 0,
       .usage = "Suppress printing of header line",
     },
+    { .name = "pending", .key = 'p', .has_arg = 0,
+      .usage = "List pending jobs",
+    },
+    { .name = "running", .key = 'r', .has_arg = 0,
+      .usage = "List running jobs",
+    },
+    { .name = "inactive", .key = 'i', .has_arg = 0,
+      .usage = "List inactive jobs",
+    },
+    { .name = "userid", .key = 'u', .has_arg = 1,
+      .usage = "Limit output to specific userid. " \
```

Comment:
Minor: might want to add a `.arginfo = "UID"` or something here so that `--userid` shows as taking an option argument in `--help` output.

Reply:
good idea